pkg/gameserverallocations/controller_test.go

```diff
@@ -60,6 +60,11 @@ const (
 func TestControllerAllocator(t *testing.T) {
 	t.Parallel()
 
+	// TODO:(markmandel) remove once GameServerSelector is in place, since right now enabling the feature flag causes flaky tests
```

Comment:
This comment is a bit scary. Does it make the tests flaky because they sometime run with the feature flag on and sometimes with it off?

Reply:
That is understandably scary on review 😃 - let me adjust it so it is less scary.

TL;DR: once all the code is in place the test will pass with any feature flag enabled/disabled - but not yet.